src/components/SortedInformation/SortedInformation.component.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+import { SortedInformatonContainer } from './SortedInformation.styles';
+
+const SortedInformation = ({ departure, arrival }) => {
+	return (
+		<SortedInformatonContainer>
+			<div className="information-container">
+				<span className="tag">가는편</span>
+				<div className="departure-and-arrival">
+					<span>{departure}</span>
+					<i className="fas fa-long-arrow-alt-right" />
+					<span>{arrival}</span>
+				</div>
+				<span>11월03일(화)</span>
+			</div>
+		</SortedInformatonContainer>
+	);
+};
+
+const mapStateToProps = ({ airTickets: { departure, arrival } }) => ({
+	departure,
+	arrival,
+});
```

Comment:
```suggestion
const mapStateToProps = ({ airTickets }) => ({
	departure: airTickets.departure,
	arrival: airTickets.arrival,
});
```
2 depth 이상 구조분해 할당은 가독성 측면에서 좋지 않은 것 같습니다

Reply:
수정했습니다 😄